README.md

```diff
@@ -6,23 +6,35 @@
 # CNTO Incident Detection
 This repo contains the scripts used by [Carpe Noctem Tactical Operations](http://www.carpenoctem.co) to monitor the status of its online services.
 
-Every script is compliant to the Nagios standards in order to be attached to a Nagios based monitoring system.
+Every script is compliant to the Nagios Plugins standards in order to be attached to a Nagios based monitoring system.
 
 ## Requirements
  - Python 3.5
 
 ## Installation
 It is recommended to install the package under a virtual environment (for example `virtualenv`).
 
-Installation is possible via `pip`: clone or download the repository, open a terminal window and change location to the project main folder then run
-`pip install .`
-
-## Available scripts
+Installation is possible via `pip`: clone or download the repository, open a terminal window and change location to the project main folder then run `pip install .`
+
+## Usage
+ The `cnto-check-runner` script executes a program which must abide to Nagios Plugins guidelines (see
+http://nagios-plugins.org/doc/guidelines.html) and updates a single component on a configured
+instance of Cachet to reflect the status detected by the script. If the status detected is `WARNING`
+or `CRITICAL` the runner will execute the monitoring program again for a maximum number of times
+given as argument and waiting a given amount of time between each attempt. Cachet will be updated
+only if the status remains either `WARNING` or `CRITICAL` after all the retries. As of this version,
+the runner makes no difference between `WARNING` and `CRITICAL` status, the component will be set as
```

Comment:
That's different from what we discussed earlier. You mentioned on Trello: 
```
WARNING: Partial Outage
CRITICAL: Major Outage
```

What proved too costly to have that feature in this iteration?